src/mod/applications/mod_http_cache/mod_http_cache.c

```diff
@@ -1206,7 +1206,10 @@ static switch_status_t http_get(url_cache_t *cache, http_profile_t *profile, cac
 		
 		for (i = 0; i < cache->max_retry; ++i) {
 			curl_status = switch_curl_easy_perform(curl_handle);
```

Comment:
I think the default download timeout is 3sec. Don't we need to adjust the timeout sec to 5?

Reply:
the download timeout is configurable and set on the b2bua configuration e.g. http_cache.conf.xml. Actually the default download timeout is 300 seconds. The 3 integer value default value I think you are referring to the default max retry

Reply:
Ah, right.

But then, don't we need to adjust this to a certain sec anyway? Because it will wait for 300 sec. The ticket is mentioning the expecting timeout is 5 sec.